src/classes/i18n.js

```diff
@@ -183,13 +226,26 @@ export default class i18n {
             this.languages.push(default_lang);
         }
 
-        return new Promise(resolve => {
+        await new Promise(resolve => {
             async.each(
                 this.languages,
                 this.get_lang_files.bind(this),
                 this.load_files.bind(this, resolve)
             );
         });
+
+        try {
+            // save version to VERSION file
+            return await fs_promises.writeFile(
+                this.current_version_path,
+                service_version
+            );
+
```

Comment:
```suggestion
        // save LANGUAGE file
        await fs_promises.writeFile(
            this.languages_path,
            JSON.stringify(languages)
        );
```

Reply:
Actually, I don't think we need this at all since a proper VERSION file already indicates properly saved language files?

Reply:
VERSION simply saves the version of the downloaded translation. We still need this to cache the response of fetching the languages. This contains
```
{"data":{"languages":["ru","nl","de","zh_TW","th","hi","fr","pt","ar","af","es","sq"]}}

```

Reply:
Let's save everything in one JSON file then to make things simple? If we have a proper file saved, it means we have the proper files downloaded? (Otherwise the download would already throw exception?)